src/main/java/at/hannibal2/skyhanni/utils/ComputerTimeOffset.kt

```diff
@@ -0,0 +1,124 @@
+package at.hannibal2.skyhanni.utils
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.DebugDataCollectEvent
+import at.hannibal2.skyhanni.events.ProfileJoinEvent
+import at.hannibal2.skyhanni.events.SecondPassedEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.test.command.ErrorManager
+import at.hannibal2.skyhanni.utils.TimeUtils.format
+import kotlinx.coroutines.launch
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import org.apache.commons.net.ntp.NTPUDPClient
+import java.net.InetAddress
+import kotlin.concurrent.thread
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.Duration.Companion.seconds
+
+@SkyHanniModule
+object ComputerTimeOffset {
+    var offsetMillis: Duration? = null
+        private set
```

Comment:
i dont think this was ever used anywhere else so it can probably be a regular private variable unless there are still plans for this offset to be integrated elsewhere into the mod